promptsource/promptsource.py

```diff
@@ -196,15 +204,14 @@ def save_data(message="Done!"):
                 index = template_list.index(new_template_name)
             else:
                 index = 0
-            template_name = st.selectbox('', template_list, key='template_select',
-                                          index=index, help='Select the template to work on.')
+            template_name = st.selectbox(
+                "", template_list, key="template_select", index=index, help="Select the template to work on."
+            )
 
             if st.button("Delete Template", key="delete_template"):
-                templates.remove_template(template_key, template_name)
+                templates.remove_template(template_key, template.get_name())
```

Comment:
Looks like this got reverted back on this branch. It should indeed be `template_name` because the object doesn't need to be instantiated.

Reply:
Oops, I probably chose wrong side when rebase-force-push. Let me try it again. Apologies for the trouble.

Reply:
Seems fixed.